src/test/index.test.ts

```diff
@@ -208,6 +209,48 @@ describe('Typegoose', () => {
     expect(newfound.virtualSubs[1]._id.toString()).to.be.equal(virtualsub3._id.toString());
     expect(newfound.virtualSubs).to.not.include(virtualsub2);
   });
+
+  it('Should support dynamic references via refPath', async () => {
+    const sprite = new Beverage({
+        isDecaf: true,
+        isSugarFree: false
+    })
```

Comment:
could you add `;` to the line endings (all in the file) to fit to fit the others?